grails-app/views/wps/show.gsp

```diff
@@ -40,10 +40,10 @@
                            please let us
                            know.
                            All feedback is very welcome. For help and information about this site
-                           please contact <a href="mailto:info@aodn.org.au">info@aodn.org.au</a></p>
+                           please contact <a href="mailto:${grailsApplication.config.portal.contactEmail}">${grailsApplication.config.portal.contactEmail}</a></p>
                     </div>
                     <div class="col-md-8">
-                        <p>Use of this web site and information available from it is subject to our <a href="http://imos.org.au/imostermsofuse0.html">
+                        <p>Use of this web site and information available from it is subject to our <a href="${grailsApplication.config.portal.conditionOfUse}">
                            Conditions of use
                         </a></p>
                         <p>© 2016 IMOS</p>
```

Comment:
Sorry I didnt spot this. Maybe remove the whole `<p>© 2016 IMOS</p>`
We dont even have copyright on the landing page
@bhasin85